CMakeLists.txt

```diff
@@ -121,15 +126,55 @@ endif()
 
 # Install the library (relative to the CMAKE_INSTALL_PREFIX)
 # ----------------------------------------------------------
-install(TARGETS gqcp EXPORT gqcpTargets
+
+include(GNUInstallDirs)
+
+install(TARGETS gqcp
+        EXPORT gqcp-targets
         LIBRARY DESTINATION lib
         ARCHIVE DESTINATION lib
         RUNTIME DESTINATION bin
         INCLUDES DESTINATION include
         )
 
-install(EXPORT gqcpTargets
-        FILE gqcpTargets.cmake
+install(DIRECTORY ${CMAKE_BINARY_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+install(EXPORT gqcp-targets
+        FILE gqcp-targets.cmake
         NAMESPACE gqcp::
-        DESTINATION lib/cmake/gqcp
+        DESTINATION cmake
         )
+
+# Configure package
```

Comment:
Since we're making a `gqcp-config.cmake` file, this means that any downstream users should use `find_package(gqcp NO_MODULE)`, right?

Could you add this to the README.md as well?